"""
show each candidate and their scores 1 by 1 then the second h1 heading then the votes count then show winner
should be able to change picture and name, based on who wins by votes
retreive and interpret votes from file
"""
from PyQt5 import QtCore, QtGui, QtWidgets
from candidatemenu import *

class Ui_Results(object):
    def setupUi(self, Results):
        Results.setObjectName("Results")
        Results.resize(703, 797)
        Results.setMinimumSize(QtCore.QSize(703, 797))
        Results.setMaximumSize(QtCore.QSize(703, 797))
        self.label_15 = QtWidgets.QLabel(Results)
        self.label_15.setGeometry(QtCore.QRect(170, 280, 391, 141))
        font = QtGui.QFont()
        font.setPointSize(22)
        self.label_15.setFont(font)
        self.label_15.setWordWrap(True)
        self.label_15.setObjectName("label_15")
        self.label_13 = QtWidgets.QLabel(Results)
        self.label_13.setGeometry(QtCore.QRect(50, 110, 141, 151))
        self.label_13.setText("")
        self.label_13.setPixmap(QtGui.QPixmap("candidate1.jpg"))
        self.label_13.setScaledContents(True)
        self.label_13.setObjectName("label_13")
        self.label_14 = QtWidgets.QLabel(Results)
        self.label_14.setGeometry(QtCore.QRect(260, 110, 141, 151))
        self.label_14.setText("")
        self.label_14.setPixmap(QtGui.QPixmap("candidate3.jpg"))
        self.label_14.setScaledContents(True)
        self.label_14.setObjectName("label_14")
        self.label_16 = QtWidgets.QLabel(Results)
        self.label_16.setGeometry(QtCore.QRect(470, 110, 141, 151))
        self.label_16.setText("")
        self.label_16.setPixmap(QtGui.QPixmap("candidate2.jpg"))
        self.label_16.setScaledContents(True)
        self.label_16.setObjectName("label_16")
        self.label_17 = QtWidgets.QLabel(Results)
        self.label_17.setGeometry(QtCore.QRect(170, -10, 391, 141))
        font = QtGui.QFont()
        font.setPointSize(22)
        self.label_17.setFont(font)
        self.label_17.setWordWrap(True)
        self.label_17.setObjectName("label_17")
        self.label_18 = QtWidgets.QLabel(Results)
        self.label_18.setGeometry(QtCore.QRect(110, 290, 21, 21))
        font = QtGui.QFont()
        font.setPointSize(14)
        self.label_18.setFont(font)
        self.label_18.setWordWrap(True)
        self.label_18.setObjectName("label_18")
        self.label_19 = QtWidgets.QLabel(Results)
        self.label_19.setGeometry(QtCore.QRect(320, 290, 21, 21))
        font = QtGui.QFont()
        font.setPointSize(14)
        self.label_19.setFont(font)
        self.label_19.setWordWrap(True)
        self.label_19.setObjectName("label_19")
        self.label_20 = QtWidgets.QLabel(Results)
        self.label_20.setGeometry(QtCore.QRect(530, 290, 21, 21))
        font = QtGui.QFont()
        font.setPointSize(14)
        self.label_20.setFont(font)
        self.label_20.setWordWrap(True)
        self.label_20.setObjectName("label_20")
        self.label_21 = QtWidgets.QLabel(Results)
        self.label_21.setGeometry(QtCore.QRect(190, 390, 281, 271))
        self.label_21.setText("")
        self.label_21.setPixmap(QtGui.QPixmap("candidate2.jpg"))
        self.label_21.setScaledContents(True)
        self.label_21.setObjectName("label_21")
        self.label_22 = QtWidgets.QLabel(Results)
        self.label_22.setGeometry(QtCore.QRect(240, 640, 201, 91))
        font = QtGui.QFont()
        font.setPointSize(14)
        self.label_22.setFont(font)
        self.label_22.setWordWrap(True)
        self.label_22.setObjectName("label_22")
        self.pushButton_8 = QtWidgets.QPushButton(Results)
        self.pushButton_8.setGeometry(QtCore.QRect(110, 720, 200, 50))
        self.pushButton_8.setObjectName("pushButton_8")
        self.pushButton_9 = QtWidgets.QPushButton(Results)
        self.pushButton_9.setGeometry(QtCore.QRect(360, 720, 200, 50))
        self.pushButton_9.setObjectName("pushButton_9")

        self.retranslateUi(Results)
        QtCore.QMetaObject.connectSlotsByName(Results)

    def retranslateUi(self, Results):
        _translate = QtCore.QCoreApplication.translate
        Results.setWindowTitle(_translate("Results", "Voting Machine"))
        self.label_15.setText(_translate("Results", "And the winner is..."))
        self.label_17.setText(_translate("Results", "The results are in..."))
        self.label_18.setText(_translate("Results", "1"))
        self.label_19.setText(_translate("Results", "2"))
        self.label_20.setText(_translate("Results", "3"))
        self.label_22.setText(_translate("Results", "Patricia A. Spiegel"))
        self.pushButton_8.setText(_translate("Results", "Start Another Election"))
        self.pushButton_9.setText(_translate("Results", "Close Program"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Results = QtWidgets.QDialog()
    ui = Ui_Results()
    ui.setupUi(Results)
    Results.show()
    sys.exit(app.exec_())
